doc/source/release-notes/Percona-Server-5.7.21-20.rst

```diff
@@ -0,0 +1,114 @@
+.. rn:: 5.7.21-20
+
+========================
+Percona Server 5.7.21-20
+========================
+
+Percona is glad to announce the release of Percona Server 5.7.21-20
+on February 15, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.21-20/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.21
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-21.html>`_
+and includes all the bug fixes in it.
+Percona Server 5.7.21-20 is now the current GA (Generally Available) release
+in the 5.7 series. All software developed by Percona is open-source and free.
+
+New Features
+============
+
+* A new string variable :variable:`version_suffix` allows to change suffix
+  for the |Percona Server| version string returned by the read-only
+  :variable:`version` variable. Also :variable:`version_comment` is converted
+  from a global read-only to a global read-write variable.
+
+Bugs Fixed
+==========
+
+* mysqld startup script was unable to detect jemalloc library location for
+  preloading, and that prevented starting |Percona Server| on systemd based
+  machines. Bugs fixed :psbug:`3784` and :psbug:`3791`.
+
+* There was a problem with fulltext search, which could find a word with
+  punctuation marks in natural language mode only, but not in boolean mode.
+  Bugs fixed :psbug:`258`, :psbug:`2501` (upstream :mysqlbug:`86164`).
+
+* A build errors were present on FreeBSD (caused by fixing the bug
+  :psbug:`255`) and on MacOS (caused by fixing the bug :psbug:`264`).
+  Bugs fixed :psbug:`2284` and :psbug:`2286`.
+
+* A bunch of fixes where introduced to make |Percona Server| compile under
+  GCC 7. Bugs fixed :psbug:`3780` (upstream :mysqlbug:`89420`,
+  :mysqlbug:`89421`, and :mysqlbug:`89422`). 
+
+* CMake error took place at compilation with bundled zlib. Bug fixed 
+  :psbug:`302`.
+
+* A GCC 7 warning fix introduced regression in |Percona Server| that lead to
+  a wrong SQL query built to access the remote server when Federated storage
+  engine was used. Bug fixed :psbug:`1134`. 
+
+* It was possible to enable ``encrypt_binlog`` with no binary logging. Bug
+  fixed :psbug:`287`.
+
+* Long buffer wait times where occuring on busy servers in case of several
+  parallel workers. Bug fixed :psbug:`276`.
+
+* Server queries that contained JSON special characters and were logged by
+  ``audit_log`` plugin in JSON format caused invalid output due to lack of
+  escaping. Bug fixed :psbug:`1115`.
+
+* Percona Server now uses *Travis CI*  for additional tests. Bug fixed
+  :psbug:`3777`.
+
+Other bugs fixed: :psbug:`257`, :psbug:`1090` (upstream :mysqlbug:`78048`), :psbug:`1109`, :psbug:`1127`, :psbug:`2415`,
+:psbug:`3767`, and :psbug:`3804` (upstream mysqlbug:`89598`).
+
```

Comment:
only 21 entry, while milestone contains 34, some bugs (like ps-1075 and ps-3781) are missing from the list.